maxim/tests/test_test_runs.py

```
@@ -14,17 +14,20 @@
 from maxim.models.evaluator import (
     PassFailCriteriaForTestrunOverall,
     PassFailCriteriaOnEachEntry,
 )
 
-with open(str(f"{os.getcwd()}/libs/maxim-py/maxim/tests/testConfig.json")) as f:
+with open(str(f"{os.getcwd()}/maxim/tests/testConfig.json")) as f:
     data = json.load(f)
 
 logging.basicConfig(level=logging.INFO)
-env = "dev"
+env = "prod"
```

Pylint (3.3.7)

[convention] Constant name "env" doesn't conform to UPPER_CASE naming style

(C0103)